File: NoteBase.Core/DTOs/NoteTagsDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NoteBase.Core.DTOs
{
    public class NoteTagsDto
    {
        public int Id { get; set; }
        public int TagId { get; set; }
        public int NoteId { get; set; }

    }
}

File: NoteBase.Android.Java/NoteBaseAndroidCore/src/main/java/com/veyselkaranitekgoz/notebaseandroid/core/DTOs/NoteTagsDto.java
package com.veyselkaranitekgoz.notebaseandroid.core.DTOs;

public class NoteTagsDto {
    private int Id;
    private int TagId;
    private int NoteId;

    public int getId() {
        return Id;
    }

    public void setId(int id) {
        Id = id;
    }

    public int getTagId() {
        return TagId;
    }

    public void setTagId(int tagId) {
        TagId = tagId;
    }

    public int getNoteId() {
        return NoteId;
    }

    public void setNoteId(int noteId) {
        NoteId = noteId;
    }
}
